### DISCUSSION TITLE
TO5GR logged as Not Found

### DISCUSSION THREAD

**Discussion by ve3hls** [magicbug/Cloudlog#1086] [General]
I uploaded my adif file from the IARU HF Championship to Cloudlog and TO5GR was listed as Not Found.  TO5GR is in Martinique.

73,

Ken  VE3HLS

**Comment by magicbug** [comment]
These are special calls in theory, If you go to admin -> update the country files and click "update dxcc data" then wait till you see it say the word "Done" and recheck your log in the same area should resolve it, as Clublog data has an exception for it (tested here)

I'd suggest maybe updating it after a contest or so often.

Pete.
